api/envoy/extensions/filters/http/stateful_session/v3/stateful_session.proto

```diff
@@ -23,6 +23,10 @@ message StatefulSession {
   //
   // [#extension-category: envoy.http.stateful_session]
   config.core.v3.TypedExtensionConfig session_state = 1;
+
+  // If set to True, the HTTP request must be routed to the requested destination.
+  // If the requested destination is not available, Envoy returns 503. Defaults to False.
```

Comment:
Just FYI, note that if/when we implement this feature in gRPC, we would return status UNAVAILABLE instead of an HTTP status code.  But [HTTP 503 maps to UNAVAILABLE](https://github.com/grpc/grpc/blob/master/doc/http-grpc-status-mapping.md) anyway, so we don't necessarily need to explicitly note this here.

Reply:
In gRPC, the SSA feature operates on the client side, not the server side, so there is no HTTP code involved; it would simply report to the application that the RPC failed with status UNAVAILABLE.

If someone used gRPC with an Envoy sidecar proxy and the SSA feature was used in Envoy instead of in gRPC, then Envoy would return HTTP status 503, which the gRPC client would map to status UNAVAILABLE.  So the client application would see the same result in both cases.

Reply:
Thanks for explaining!